Libplanet.Explorer/GraphTypes/AccountStateType.cs

```diff
@@ -83,112 +82,6 @@ public AccountStateType()
                         .Select(address => context.Source.GetState(address))
                         .ToArray()
             );
-
-            Field<IValueType>(
```

Comment:
Maybe we should check ok to remove those types. Previously, we had a problem as I remember...

Reply:
Arg... you are right. I'll revert it back.
However, the solution to this predicament can't be "just keep adding stuff". 😕

Reply:
Reverted.